assets/src/data/services/apollo/queries/generalSettings/useGeneralSettings.ts

```diff
@@ -1,4 +1,4 @@
-import { GET_GENERAL_SETTINGS } from './';
+import { GET_GENERAL_SETTINGS } from '.';
```

Comment:
is this smth OS-related?

Reply:
Something `VS Code` related 🤦‍♂️ 😄 
I let it update the paths